Doc/library/sqlite3.rst

```diff
@@ -601,25 +601,16 @@ Connection objects
 
       Example:
 
-      .. testcode::
-
-         def dict_factory(cursor, row):
-             d = {}
-             for idx, col in enumerate(cursor.description):
-                 d[col[0]] = row[idx]
-             return d
-
-         con = sqlite3.connect(":memory:")
-         con.row_factory = dict_factory
-         cur = con.execute("SELECT 1 AS a")
-         print(cur.fetchone()["a"])
-
-         con.close()
-
-      .. testoutput::
-         :hide:
+      .. doctest::
 
-         1
+         >>> def dict_factory(cursor, row):
+         ...     col_names = [col[0] for col in cursor.description]
+         ...     return {key: value for key, value in zip(col_names, row)}
+         >>> con = sqlite3.connect(":memory:")
+         >>> con.row_factory = dict_factory
+         >>> for row in con.execute("SELECT 1 as a, 2 as b"):
```

Comment:
```suggestion
         >>> for row in con.execute("SELECT 1 AS a, 2 AS b"):
```

Minor nit, but per the style guide above and the original example, should `AS` be a capitalized keyword?

Reply:
Yeah, thanks for catching!